girder/utility/resource.py

```diff
@@ -11,7 +11,6 @@ def _walkTree(node, path=[]):
             full_path = list(path)
             full_path.append(k)
             routeMap[v] = full_path
-            path = []
```

Comment:
Just noticed this is a mutable default var... maybe we should change it from `[]` to `()`?

Reply:
Good plan, done